docs/README.md

```diff
@@ -1,12 +1,14 @@
 # Overview
 
-IoT Identity Service is made up of three services that provide provisioning and cryptographic services:
+Azure IoT Edge brings the analytical power of the cloud closer to devices to drive better business insights and enable offline decision making through the use of [Edge modules](https://docs.microsoft.com/azure/iot-edge/iot-edge-modules). The full IoT Edge runtime, in concert with IoT Hub, [provides numerous capabilities](https://docs.microsoft.com/azure/iot-edge/iot-edge-runtime) to scale out and manage an IoT solution from the cloud.
+
+The IoT Identity Service is one component of IoT Edge and can be used stand-alone. The "Identity Service" is technically made up of three services (purple in the diagram) that provide provisioning and cryptographic services. They provide the foundation for authorized components on the device to communicate with Azure services (i.e. IoT Hub) whether as the [device](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry) or an individual [module](https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-module-twins). Containerized modules (i.e. IoT Edge modules) require the full IoT Edge runtime. Non-containerized modules (i.e. host-level modules) minimally need the identity, cert, and key services described below.
 
 - [Identity Service](identity-service.md)
 
     This service provisions the device identity and module identities with Azure. 
 
-    The Identity Service provides access to host-level processes and container modules to connect to the cloud using tokens or X.509 certificate credentials, corresponding to their respective identities.
+    The Identity Service provides access to both host-level processes and container modules (via the IoT Edge runtime) to connect to the cloud using tokens or X.509 certificate credentials, corresponding to their respective identities.
```

Comment:
I would just remove the distinction between processes and containers. Let us call them all "modules". IoT Edge currently doesn't manage host-level processes (only root user can), but that could change in the future.